voter-dapp/src/ActiveRequests.js

```diff
@@ -136,25 +158,33 @@ function ActiveRequests() {
     );
   }
 
+  const hasPendingTransactions = status === "pending";
   const statusDetails = voteStatuses.map((voteStatus, index) => {
+    let currentVote = "";
+    if (voteStatus.committedValue && decryptedCommits[index].price) {
+      currentVote = web3.utils.fromWei(decryptedCommits[index].price.toString());
```

Comment:
nit: should we use the `formatWei()` function from FormattingUtils.js? Or is there no reason to use it here? I forget exactly why we added that function.

Reply:
Yep, we should. Here, and in one other place in this file.